src/components/AppComponent.js

```diff
@@ -7,30 +7,66 @@ import ImportExport from '../containers/ImportExport'
 import Help from '../components/Help'
 import ActualLanguage from '../containers/ActualLanguage';
 import { Provider } from 'react-redux'
-import { createStore } from 'redux'
+import { applyMiddleware, createStore, compose } from 'redux'
 import reducer, { exportState, importState, freshState } from '../reducers';
-import { updateAxioms, updateTheorems, updateNewTheorem } from '../actions';
+import { updateAxioms, updateTheorems, updateNewTheorem, UPDATE_PROOF, CHANGE_CONST, CHANGE_FUN, CHANGE_PRED } from '../actions';
 import LogicContext from './LogicContext'
 
 import '../static/css/bootstrap.min.iso.css';
 import '../static/css/style.iso.css';
 
+function filterAction(action) {
+  switch(action.type) {
+    case UPDATE_PROOF:
+      return true;
+    case CHANGE_CONST:
+    case CHANGE_PRED:
+    case CHANGE_FUN:
+      if (action.ignore === true) {
+        return true;
+      }
+      return false;
+    default:
+      return false;
+  }
+}
+
 function prepare(initialState) {
+  const composeEnhancers =
+  typeof window === 'object' &&
+  window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ?   
+    window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
+      name: 'resolution-editor'
+    }) : compose;
+  
+  const stateMonitor = store => next => action => {
+    console.log('resolution editor redux action:', action)
+    let notifyWorkbook = !filterAction(action);
+    if (instance.handleStoreChange) {
+      instance.handleStoreChange(notifyWorkbook);
+    }
+    return next(action)
+  }

```

Comment:
Nevyrába toto zbytočný šum v konzole? Nedalo by sa to zapnúť, povedzme, iba keď je `process.env.NODE_ENV == "development"`?

Reply:
ano, chcel som to odstranit ale nejak to tam ostalo